psi4/driver/procrouting/dft/dft_builder.py

```diff
@@ -377,7 +377,10 @@ def build_superfunctional_from_dictionary(func_dictionary, npoints, deriv, restr
     if "citation" in func_dictionary:
         sup.set_citation(func_dictionary["citation"])
     if "description" in func_dictionary:
-        sup.set_description(func_dictionary["description"])
+        if "doi" in func_dictionary:
```

Comment:
not really -- I just didn't feel like adding a new member data to `Functional`. think it should be separate? it may be a list anyways from compound fctls, so special ready-to-use as URL use is limited.